content/learn/book/migration-guides/0.9-0.10/_index.md

```diff
@@ -0,0 +1,733 @@
+weight = 6
+sort_by = "weight"
+template = "book-section.html"
+page_template = "book-section.html"
+insert_anchor_links = "right"
+[extra]
+long_title = "Migration Guide: 0.9 to 0.10"
++++
+
+Bevy relies heavily on improvements in the Rust language and compiler.
+As a result, the Minimum Supported Rust Version (MSRV) is "the latest stable release" of Rust.
+<div class="migration-guide">
+
+### [bevy_reflect: Pre-parsed paths](https://github.com/bevyengine/bevy/pull/7321)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Animation</div>
+    <div class="migration-guide-area-tag">Reflection</div>
+</div>
+
+`GetPath` methods have been renamed according to the following:
+
+- `path` -> `reflect_path`
+- `path_mut` -> `reflect_path_mut`
+- `get_path` -> `path`
+- `get_path_mut` -> `path_mut`
+
+### [Remove App::add_sub_app](https://github.com/bevyengine/bevy/pull/7290)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">App</div>
+</div>
+
+`App::add_sub_app` has been removed in favor of `App::insert_sub_app`. Use `SubApp::new` and insert it via `App::add_sub_app`
+
+Old:
+
+```rust
+let mut sub_app = App::new()
+// Build subapp here
+app.add_sub_app(MySubAppLabel, sub_app);
+```
+
+New:
+
+```rust
+let mut sub_app = App::new()
+// Build subapp here
+app.insert_sub_app(MySubAppLabel, SubApp::new(sub_app, extract_fn));
+```
+
+### [asset: make HandleUntyped::id private](https://github.com/bevyengine/bevy/pull/7076)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Assets</div>
+</div>
+
+- Instead of directly accessing the ID of a `HandleUntyped` as `handle.id`, use the new getter `handle.id()`.
+
+### [Break `CorePlugin` into `TaskPoolPlugin`, `TypeRegistrationPlugin`, `FrameCountPlugin`.](https://github.com/bevyengine/bevy/pull/7083)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Core</div>
+</div>
+
+- `CorePlugin` broken into separate plugins.  If not using `DefaultPlugins` or `MinimalPlugins` `PluginGroup`s, the replacement for `CorePlugin` is now to add `TaskPoolPlugin`, `TypeRegistrationPlugin`, and `FrameCountPlugin` to the app.
+
+### [Replace `RemovedComponents<T>` backing with `Events<Entity>`](https://github.com/bevyengine/bevy/pull/5680)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+- Add a `mut` for `removed: RemovedComponents<T>` since we are now modifying an event reader internally.
+- Iterating over removed components now requires `&mut removed_components` or `removed_components.iter()` instead of `&removed_components`.
+
+### [Remove `ExclusiveSystemParam::apply`](https://github.com/bevyengine/bevy/pull/7489)
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">ECS</div>
+</div>
+
+_Note for maintainers: this migration guide makes more sense if it’s placed above the one for #6919._

```

Comment:
This should be addressed. Probably should be merged into that PR's migration guide.

Reply:
For things like that, I prefer waiting until closer to release. Since this is generated manually, it will be really noisy to move it everytime I re-generate it.